pkgs/by-name/cr/crystal-dock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+  qt6,
+}:
+stdenv.mkDerivation rec {
```

Use of `finalAttrs` is preferred over `rec`, if this suggestion is accepted please also close the newly opened parenthesis before `finalAttrs` at the end of the file:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```